src/test/java/de/caritas/cob/agencyservice/api/admin/service/agency/DemographicsConverterTest.java

```diff
@@ -0,0 +1,62 @@
+package de.caritas.cob.agencyservice.api.admin.service.agency;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import de.caritas.cob.agencyservice.api.model.DemographicsDTO;
+import de.caritas.cob.agencyservice.api.repository.agency.Agency;
+import org.junit.jupiter.api.Test;
+
+class DemographicsConverterTest {
+
+  @Test
+  void fromAgency_Should_ConvertEmptyGenderListIfGenderIsNull() {
```

Comment:
not _When_ ? I asked Markus but it looks like we do not have a test-name-pattern.
That Should-pattern makes me crazy but currently we writing so different names in each file lol that makes me crazy

Reply:
I agree, in majority of cases we use When, so I've renamed it

Reply:
thanks. I will also take care about that.